packages/decompress/decompress.0.5/url

```diff
@@ -1,2 +1,2 @@
 http: "https://github.com/oklm-wsh/Decompress/archive/0.5.tar.gz"
-checksum: "667fdfa64423e01ef38242f305e80e7f743bc0db"
+checksum: "4bfd0e37dc9b946bf5aa195fd7e2f236"
```

Comment:
Use `topkg bistro` to enjoy your beer. There's no need to tweak these things manually.